### ISSUE TITLE
Allow for auto build of Docker files

### ISSUE THREAD

**Issue by brandonsoto** [moorem27/video_stream#6]
As of right now, auto build on [Docker Hub](https://hub.docker.com/) fails to build our Docker images. This is due to the Raspberry Pi's Dockerfile being based on an `arm` image. We need to find a way to build `arm` containers on there. 

## Resources
[Building ARM Containers on any x86 Machine.](https://resin.io/blog/building-arm-containers-on-any-x86-machine-even-dockerhub/). 
[Resin's ARM7hf github](https://github.com/resin-io-projects/armv7hf-debian-qemu/tree/524e1acd29acf46816a7ed0675a61beb34a994f7)
[Building ARM containers with Travis](https://blog.hypriot.com/post/setup-simple-ci-pipeline-for-arm-images/)

**Comment by brandonsoto** [comment]
The following `travis.yml` will build the images. The only thing is we need a way to test `C++` and build Dockerfiles through Travis. In addition, I still need to test whether the emulated docker image works on `arm` architectures.

```yaml
sudo: required

services:
- docker

language: bash

script:
- docker build -f Dockerfile-server -t brandonsoto/vsserver:latest .
# prepare qemu
- docker run --rm --privileged multiarch/qemu-user-static:register --reset
# build image
- docker build -f Dockerfile-client -t brandonsoto/vsclient:latest .
# test image
- docker run brandonsoto/vsserver
- docker run brandonsoto/vsclient

# push image
- >
  if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
    docker login -u="$DOCKER_USER" -p="$DOCKER_PASS"
    docker push brandonsoto/vsserver:latest
    docker push brandonsoto/vsclient:latest
  fi
```

**Comment by brandonsoto** [comment]
Partially fixed by #8. Unfortunately, the emulated image will not work on `arm` architectures.

**Comment by brandonsoto** [comment]
Fixed as of 2061c5db9fd7734071e8d026c275909a9f1d97b7
